tsconfig.json

```diff
@@ -1,3 +1,5 @@
+// This file is only here for usage by our IDE. The tsconfig that angular really uses is in `tsconfig.base.json`. ´
```

Comment:
This file should look different after the migration.
Update your global ng-cli and try to generate a new project, and then a library. Then this file is generated:
```json
/*
  This is a "Solution Style" tsconfig.json file, and is used by editors and TypeScript’s language server to improve development experience.
  It is not intended to be used to perform a compilation.

  To learn more about this file see: https://angular.io/config/solution-tsconfig.
*/
{
  "files": [],
  "references": [
    {
      "path": "./tsconfig.app.json"
    },
    {
      "path": "./tsconfig.spec.json"
    },
    {
      "path": "./projects/test-lib/tsconfig.lib.json"
    },
    {
      "path": "./projects/test-lib/tsconfig.spec.json"
    }
]
}



```